Board collab

vsanantone opened the pull request:
Finished Writing the test for board_spec.rb(iteration 2)

vsanantone committed:
fix: Switching branches

vsanantone committed:
fix/feat/board: Refactor the width value to string

vsanantone committed:
feat/test/board: finish collab session

vsanantone committed:
test/board: add coordinates for valid placement

vsanantone committed:
test/board: add testing for range & ordinal method

vsanantone committed:
test/board: add test for place method

vsanantone committed:
test/board: add test for render method

chisPmama commented on lib/board.rb:
```diff
@@ -25,23 +25,26 @@ def valid_coordinate?(coordinate)
-      if coordinate.delete("^0-9") || coordinates.delete("^0-9") > 
-      num+=1
-      coordinate[num] = coordinate.next
+    ship.length == coordinates.count #needs logic
```
Refactoring this today, but we can merge just to keep the ball rolling!

chisPmama committed:
update/test: Edits the testing and reorganizes sections of Iteration II testing

Small changes and included a couple other "it/do" verification methods to include other blocks of the interaction code that was missing!
